index.html

```diff
@@ -244,12 +243,9 @@ <h2>Success Criteria</h2>
       <a href="#advancement">§3.1</a> to demonstrate adequate
       <a href="https://www.w3.org/Consortium/Process/#implementation-experience">implementation experience</a>.</p>
 
-    <p>Each Recommendation-track Technical Report:</p>
+    <p><span class="todo">extend all itemized lines?</span>Each Recommendation-track Technical Report:</p>
```

Comment:
I think we should take no action here, and remove this TODO.

Reply:
removed : 9457ef0